### PR TITLE
[full-ci] Unify people&link shares into one sidebar panel

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Related Issue
- Works towards https://github.com/owncloud/web/issues/6621

There's more work to do before the linked ticket can be closed, however this PR reorganizes the components file/folder structure and makes working in parallel with little conflicts possible

### CODE REVIEW

**Review by kulmann — CHANGES_REQUESTED** [review]
Logging in as `einstein`, being a member of a space, I get an error when trying to open the `members` panel.

**Comment by kulmann** [comment]
Could you fix the code smells that sonarcloud complains about?

**Review by kulmann — APPROVED** [review]
Nice! 💪

File: packages/web-pkg/src/components/sidebar/SideBar.vue [hunk 1]
```diff
@@ -29,13 +29,13 @@
         >
           <oc-button
             v-if="!panel.default"
+            v-oc-tooltip="accessibleLabelBack"
```
[the remark this answers is not in the document]
Comment (pascalwengerter) [reply packages/web-pkg/src/components/sidebar/SideBar.vue:32]: Sorry just saw this now. I updated it according to the design mockups 😬

**Pull request merged**